src/components/toolbar/PanelPreferencesToolbar.tsx

```diff
@@ -0,0 +1,55 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { ReactNode } from 'react';
+import { FaCheck, FaTimes } from 'react-icons/fa';
+
+import { Button } from '../button/Button';
+
+interface PanelPreferencesToolbarProps {
+  children?: ReactNode;
```

Comment:
The toolbar should not accept children, and not render the entire panel container. Just the toolbar, like in your other PR.

Reply:
okey, fixed